External/SDK/VisualStudio/VS2017.bff

```diff
@@ -40,11 +41,21 @@
                 .VS2017_Version         = '14.16.27023'
             #else
                 //
-                // Failed
+                // Use Visual Studio 2019 Enterprise installation (used by GitHub Actions)
                 //
```

Comment:
This is the 2017.bff, but this refers to VS 2019 (the comment and the path)?

Reply:
That is correct.
The latest VC toolchain from VS 2017 (14.16) can be installed as part of VS 2019, in that case it goes to `C:\Program Files (x86)\Microsoft Visual Studio\2019\EditionName\VC\Tools\MSVC`. This is the way it is installed on Windows runners for GitHub Actions - VS 2019 Enterprise is installed there and toolchains from VS 2017 and VS 2015 are installed as its separate components.

Reply:
Thanks.